lib/queries/getChallenge.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function (id) {
+  return 'SELECT * FROM challenges WHERE id = ' + id;
```

Comment:
I think we should try to avoid using *. Specifying the names of the column you want to return give more information about the query.